docs/UserGuide.md

```diff
@@ -73,19 +73,55 @@ Format: `help`
 
 Adds a profile to NUScheduler.
 
-Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL`
+Format: `profile -a n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A profile can have any number of tags (including 0)
+</div>
+
+Examples:
+* `profile -a n/John Doe p/98765432 e/johnd@example.com t/friend`
+* `profile -a n/Betsy Crowe e/betsycrowe@example.com p/1234567 t/professor`
```

Comment:
Maybe you can include an example where more than one tag is used 

```suggestion
* `profile -a n/Betsy Crowe e/betsycrowe@example.com p/1234567 t/professor t/lecturer`
```

Reply:
Good idea. Great catches over all. Thanks Shawn 👍🏼.